panel/layout/base.py

```diff
@@ -785,9 +794,18 @@ class ListPanel(ListLike, Panel):
     An abstract baseclass for Panel objects with list-like children.
     """
 
-    scroll = param.Boolean(default=False, doc="""
-        Whether to add scrollbars if the content overflows the size
-        of the container.""")
+    scroll = param.ObjectSelector(
```

Comment:
I think `param.Selector` should be used here as `ObjectSelector` is soft-deprecated.